crates/turbo-tasks-fs/src/lib.rs

```diff
@@ -189,11 +113,6 @@ pub struct DiskFileSystem {
     #[turbo_tasks(debug_ignore, trace_ignore)]
     #[serde(skip)]
     watcher: Arc<DiskWatcher>,
-    /// Array of paths that should not notify invalidations.
-    /// `notify` currently doesn't support unwatching subpaths from the root,
-    /// so underlying we still watches filesystem event but only skips to
-    /// invalidate.
-    ignored_subpaths: Vec<PathBuf>,
```

Comment:
moving that into `watcher` will include it in `serde(skip)` which results in looking `ignored_subpaths` after serialization